packages/Graph/Schemas/Microsoft.Graph.User.FindUsers.schema

```diff
@@ -0,0 +1,50 @@
+﻿{
+  "$schema": "https://raw.githubusercontent.com/microsoft/botframework-sdk/master/schemas/component/component.schema",
+  "$role": "implements(Microsoft.IDialog)",
+  "type": "object",
+  "title": "Microsoft Graph - Find users",
```

Comment:
Can we change this to GetUsers to match the format of the other actions?

Reply:
Sure!